src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxBound = 100;//range random 0...100
+    private Random random = new Random();//new class instance random
+    private ColorSupplier colorSupplier = new ColorSupplier();//new class instance colorSupplier
+
+    public Ball getRandomBall() {
+        int randomNumberBall = random.nextInt(maxBound);
+        Ball ball = new Ball();
+        ball.setNumber(randomNumberBall);
```

Comment:
```suggestion
        Ball ball = new Ball();
        ball.setNumber(random.nextInt(maxBound));
```

Be consistent in your approach